samples/testing-frameworks/appium/client-side/csharp/biometric-sample-test/BitbarBiometricSample/TestBiometricsIOS.cs

```diff
@@ -0,0 +1,83 @@
+﻿using OpenQA.Selenium.Appium.iOS;
+using OpenQA.Selenium.Support.UI;
+
+namespace BitbarBiometricSample
+{
+    [TestFixture]
+    public class TestBiometricsIOS
+    {
+        private readonly String SCREENSHOT_DIRECTORY = AppDomain.CurrentDomain.BaseDirectory + "/screenshots/";
+        private RemoteWebDriver driver;
+
+        [OneTimeSetUp]
+        public void InitDriver()
+        {
+            const String AppiumHubURL = "https://appium.bitbar.com/wd/hub";
+
+            AppiumOptions capabilities = new();
+            //Platfrom dependent 
+            capabilities.AddAdditionalCapability("platformName", "iOS");
+            capabilities.AddAdditionalCapability("deviceName", "iPhone device");
+            capabilities.AddAdditionalCapability("automationName", "XCUITest");
+            //Customizable
+            capabilities.AddAdditionalCapability("bitbar_project", "C# Appium Automated Test");
+            capabilities.AddAdditionalCapability("bitbar_testrun", "iOS Run");
+            capabilities.AddAdditionalCapability("bitbar_device", "Apple iPhone 11");
+            //Required
+            capabilities.AddAdditionalCapability("bitbar_apiKey", "<YOUR_BITBAR_API_KEY>");
```

Comment:
What do you think about extracting caps to another file? Especially `bitbar_apiKey` and `bitbar_app`